rootfs/opt/fluentd/deis-output/lib/fluent/plugin/out_deis.rb

```diff
@@ -26,6 +26,8 @@ def initialize
       @influx_nsq = nil
       @log_topic = ENV['NSQ_LOG_TOPIC'] || "logs"
       @metric_topic = ENV['NSQ_METRIC_TOPIC'] || "metrics"
+      @send_logs_to_nsq = if ENV['SEND_LOGS_TO_NSQ'].to_s.downcase == 'false' then false else true end
```

Comment:
Sorry was on my mobile when I typed my earlier comment. This is a better way to handle this - 

`@send_logs_to_nsq = ENV['SEND_LOGS_TO_NSQ'].to_s.downcase == 'false' ? false : true`